pkg/systemd/terminal.jsx

```diff
@@ -186,6 +263,30 @@ const _ = cockpit.gettext;
                             </ToolbarContent>
                         </Toolbar>
                     </div>
+                <div className={"warn"}>
```

Comment:
Does this class do anything? I don't see it referenced anywhere. But indeed the alert will need some styling, it should at least have some margin to the terminal and the toolbar. If you plan to keep it, please make it a bit more unique, like `ct-terminal-dir-alert`